packer/vhd-image-builder.json

```diff
@@ -30,7 +30,9 @@
             "image_sku": "{{user `ubuntu_sku`}}-LTS",
             "image_version": "latest",
             "azure_tags": {
-                "dept": "Azure Container Service"
+                "os": "Linux",
+                "now": "{{user `create_time`}}",
```

Comment:
can you use create_time on line 25 as well?
"capture_name_prefix": "aksengine-{{timestamp}}", -> "capture_name_prefix": "aksengine-{{user `create_time`}}",

Reply:
thanks!